integrationTests/testProcessorNodeWithMultisigner.go

```diff
@@ -240,6 +242,7 @@ func CreateNode(
 		keyIndex,
 		nil,
 		&mock.HeaderSigVerifierStub{},
+		&mock.HeaderIntegrityVerifierStub{},
```

Comment:
maybe you can have just one component HeaderVerifier  that check header signatures and also header integrity because now you have 2 components that verify the header.

Reply:
I will leave a TODO and maybe we can refactor in a future PR